### DISCUSSION TITLE
Version 1.0.0 changes

### DISCUSSION THREAD

**Discussion by systemcatch** [crsmithdev/arrow#923] [General]
See #832 for a complete list of changes in version 1.0.0 which has been a long time coming :tada: 

Feel free to discuss or ask any questions here.

**Comment by flatiron32** [comment]
The documentation still says that `arrow.get(None)` returns the current time UTC.

**Reply by jadchaar** [reply]
Thanks for letting us know. This will be fixed in https://github.com/arrow-py/arrow/pull/939.

**Comment by K3UL** [comment]
Hello,
The changelog for 1.0 states that Arrow now supports Python 3.9 and PEP484 type annotations. What does it mean exactly ? Are there some examples ?

I have been using Arrow with Python 3.9 and annotations with version 0.17, without a problem so I am curious about this :)

**Reply by jadchaar** [reply]
Great question! Arrow could be used without any issue prior to v1.0 with a type annotation-based Python project, but uses and invocations of Arrow-based objects and types would not be checked by type checkers like [mypy](https://mypy.readthedocs.io/en/stable/). Adding the PEP 484-style annotations to Arrow allows for type-checkers to statically analyze your uses of Arrow objects and types in your codebase to ensure that the correct types are being passed in and used throughout your code. So if you for example pass a type `None` to an Arrow function that does not support nullable types as input, mypy would throw a type-checking error so that the incompatibility can be fixed before it causes unintentional exceptions during execution :).

Thanks for using Arrow and we appreciate the question :D.

**Reply by K3UL** [reply]
Ah I see, it is about typing the Arrow codebase, I had a feeling it was something like that. I did see some new errors popping up in my checker (I use PyRight), especially a nice one telling me that I might be doing `arrow.get(None)` instead of a string, so I see what you mean.

Thank you very much for the quick answer (and for a great tool) !
